### PR TITLE
Display mission details in terminal UI

### LINKED ISSUE #39: Add mission details in Terminal
After accepting a mission in the notice board, there's no way to look at the mission's details.

The details should be added to the missions panel in the office terminal

### PR DESCRIPTION
**Pull request by maxwellpark**
Spare me the deets.

### CODE REVIEW

File: Assets/Prefabs/UI/OfficeTerminal/Missions/MissionsPanel.prefab [hunk 1]
```diff
@@ -33,7 +33,7 @@ RectTransform:
   m_RootOrder: 0
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
   m_AnchorMin: {x: 0, y: 0}
-  m_AnchorMax: {x: 0, y: 0}
```
Comment (maxwellpark) [inline Assets/Prefabs/UI/OfficeTerminal/Missions/MissionsPanel.prefab:36]: These prefabs, I swear the positioning I didn't touch. At least they look the same in-game. I added reference to the new script so needed to commit these bastards.

File: Assets/Scripts/Extensions/RectTransformExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        self.anchorMax = anchors.Item2;
+    }
+
+    public static void Stretch(this RectTransform self)
+    {
+        self.anchorMin = Vector2.zero;
+        self.anchorMax = Vector2.one;
+    }
+
+    public static void Reset(this RectTransform self)
```
Comment (maxwellpark) [inline Assets/Scripts/Extensions/RectTransformExtensions.cs:17]: The rect transform position/scale is always fucked when you create them at runtime. So this just corrects em. Might come in handy later anyway.

File: Assets/Scripts/Extensions/RectTransformExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+
+public static class RectTransformExtensions
+{
+    public static void SetAnchors(this RectTransform self, System.ValueTuple<Vector2, Vector2> anchors)
```
Comment (ambid17) [inline Assets/Scripts/Extensions/RectTransformExtensions.cs:5]: Getting spicy with the C# knowledge I see!

File: Assets/Scripts/Extensions/RectTransformExtensions.cs (hunk 2 above)
Comment (ambid17) [reply Assets/Scripts/Extensions/RectTransformExtensions.cs:17]: Good idea

**Review by ambid17 — APPROVED** [review]

**Pull request merged**